packages/mp/src/mp/core/data_models/action/parameter.py

```diff
@@ -133,7 +135,7 @@ def _from_built(cls, built: BuiltActionParameter) -> ActionParameter:
 
         """
         return cls(
-            description=built["Description"],
+            description=built.get("description") or "",
```

Comment:
Is the key name change is on purpose?

Why not
```suggestion
            description=built.get("description", ""),
```